File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.go
func arrayStringsAreEqual(word1 []string, word2 []string) bool {
    var s1,s2 string
    for i := range word1 {
        s1 = s1 + word1[i]
    }
    for i := range word2 {
        s2 = s2 + word2[i]
    }
    return s1 == s2
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String string1 = Solution.concatStrings(word1);
        String string2 = Solution.concatStrings(word2);
        return string1.equals(string2);
    }

    static public String concatStrings(String[] words) {
        String answer = "";
        for (String word: words) {
            answer += word;
        }
        return answer;
    }
}